src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -62,3 +62,4 @@ export const MAP_TYPE = {
 export const TRADITIONAL_OCL_HOST = urlConfig.TRADITIONAL_OCL_HOST;
 
 export const CUSTOM_SOURCE = 'Custom';
+export const SOURCE = 'source';
```

Comment:
Can you use a more descriptive name? What does SOURCE mean?

Reply:
I have given the constant a more descriptive name.